ksql-engine/src/test/java/io/confluent/ksql/integration/UdfIntTest.java

```diff
@@ -73,11 +91,40 @@ public void testApplyUdfsToColumns() throws Exception {
 
     Map<String, GenericRow> expectedResults = Collections.singletonMap("8", new GenericRow(Arrays.asList(null, null, "ITEM_8", 800.0, 1110.0, 12.0, true)));
 
-    Map<String, GenericRow> results = testHarness.consumeData(testStreamName, resultSchema, 4, new StringDeserializer(), IntegrationTestHarness.RESULTS_POLL_MAX_TIME_MS);
+    Map<String, GenericRow> results = testHarness.consumeData(testStreamName, resultSchema, 4,
+                                                              new StringDeserializer(),
+                                                              IntegrationTestHarness
+                                                                  .RESULTS_POLL_MAX_TIME_MS,
+                                                              DataSource.DataSourceSerDe.JSON.name());
 
     assertThat(results, equalTo(expectedResults));
   }
 
+  @Test
+  public void testApplyUdfsToColumnsAvro() throws Exception {
```

Comment:
Same as for the other test, we should be able to parameterize this and not duplicate the code.